### DISCUSSION TITLE
plots i am not creating appearing on temp drives

### DISCUSSION THREAD

**Discussion by jrnichols1974** [swar/Swar-Chia-Plot-Manager#1264] [General]
Posted this in issues as well... but maybe better for here....Trying to find out why I am having plots show up in my temp drives that I am not creating. It's filling up my temp drives and not allowing my plots to finish. I have to delete all the temp plot data from the plot id's i am not creating to get my actual plots to finish.

**Comment by Chilli71** [comment]
Take a look in the plot log files. The are in the log directory that you spezified in the manager config.
If a plot is finished but stuck in temp you can manually copy it to final and remove .2.temp from the end of the plot name.

**Comment by jrnichols1974** [comment]
These are not pools that the plot manager is creating... they are not in the logs and do not show they are being created in the manager view. I checked and made sure the is only happening when the manager is running and that is the case.  The pool IDs appear nowhere in the logs or manager.. they just start populating the temp drives. These are all pool plots... in case that matters....


![Capture](https://user-images.githubusercontent.com/20176494/126769696-1e8e5722-5408-45c6-b2c1-bd3f4026cc11.PNG)
![Capture2](https://user-images.githubusercontent.com/20176494/126769699-30aa54c0-97fd-4e27-adf3-9fbe4bfe6dc3.PNG)

**Reply by Chilli71** [reply]
I'm still not getting it.
They are created in the manager view and they are not created by the manager is a contradiction in one sentence.
Are you using MadMax?
As long as I use Swar plot manager with chia plotting I have the swar output in the debug.log and the individual chia plot outputs in the specified directory.

**Comment by jrnichols1974** [comment]
they are not in the manager and not in the logs.. they only appear as plot ids on the temp drives. from the pics I attached.. you can see the 3 plots that are being created by the manager and you can see that there is a plot on the temp drive that does not appear on the manager.. and it is also not being logged. the plot that is beginning 73d41808 is not in manager view and is not in logs. no.. not using madmax

**Reply by Chilli71** [reply]
I don't know how familiar you are with chia so please excuse if I bother you with trivial things but I need to doublecheck:
You are aware that for each plot files for each bucket are created? That means 128 buckets = 128 temporary files per plot.
Once the plot finishes all temporary files are removed.
The temporary files together can have a bit more than double the size of the final plot.

**Reply by jrnichols1974** [reply]
Yes. I understand the process. I should have included that I  only creating 1 plot per drive as I am using SAS drives for plotting. As you can see from the screen captures, there are additional plots that are appearing on the drives. Currently in this system I only have 3 SAS drives connected and plotting. I am only creating 1 plot per drive until phase 4.. about 96%. However, these other plots appear on 2 of the 3 drives but not on one. These plots are not created in swar.. are not being logged and I have no idea what is creating them. You can see from the manager view that those plot IDs are not in the manager view but they are being populated on the drives. The potting stops because the unidentified plots fill up the space on the SAS drives. If I delete the unidentified plots, plotting continues on the plots that have been created by swar. There are no other plotters running. Only swar.  Please note the plot IDs on the hardrrive screen capture and note that one of those plot IDs is not appearing in the plot manager.

**Reply by Chilli71** [reply]
In the system manager how many chia processes are kicked off?
Can you post the config file?
I have 16 temp and 16 temp2 drives and it worked until recently.
Chilli

**Comment by jrnichols1974** [comment]
[config.txt](https://github.com/swar/Swar-Chia-Plot-Manager/files/6870928/config.txt)

Of course I have the farmer public and pool contract completed... this is only running on a system with 3 drives currently...

**Reply by Chilli71** [reply]
I don't see something suspicious as this limits to three jobs with one plot each.
Is there a special reason why you start three jobs instead of defining one job with 3 temp directories? Only that way you could make use of start early.

**Reply by jrnichols1974** [reply]
start early is working this way as well...  If I put in a list of temp directories on one job, wouldn't it wait on one to finish before starting a plot on the next in the list? That must be my misunderstanding.  Do I not require a job for each drive I am plotting to? And also... if there is a problem with a plot and I need to end it.. wouldn't it be better to have different jobs for each drive so that I can kill a windows process for that job in case a problem arises?   Aside from that.. on the original issue... I stopped the manager process over night and this morning... no unrecognized plots appeared.  There is definitely something happening in the manager process causing this. Also, this only started when I updated the files to allow for pool plotting. Maybe I should try downloading everything again, replacing all files and starting over... just to see.

**Reply by Chilli71** [reply]
If you specify 3 temp drives it takes drive 1 for the first job, drive two for the second etc.
The time between the jobs is only defined by the stagger time in the job description or by the global time between jobs.
